### ISSUE TITLE
May Newsletter

### ISSUE THREAD

**Issue by RangerMauve** [datproject/newsletter#7]
Hey all, post any links you think should go in the May newsletter

Here's some stuff off the top of my head:

@cblgh's summary of some Dat stuff from DTN: https://merveilles.town/@cblgh/102141050711422037

The official DTN recap post: https://hx.ht/#post-04-dtn-recap

@sammacbeth's post about getting his Dat web extension working: https://sammacbeth.eu/blog/2019/05/12/dat-for-firefox-2.html

Livestream from the Dat event on Sunday: https://twitter.com/dat_project/status/1130094584602800129

Some visuals drawn up by @_irina for some of the talks at DTN: https://twitter.com/_lrlna/status/1129361644134567936

Cabal-desktop got fixed up by @karissa! https://github.com/cabal-club/cabal-desktop

**Comment by cblgh** [comment]
@RangerMauve new cabal desktop build at https://github.com/cabal-club/cabal-desktop/releases/tag/v2.0.0

**Comment by noffle** [comment]
Maybe a mention for [kappa-core](https://github.com/kappa-db/kappa-core)? I found out during DTN that there are now at least 4 projects building on it: [Mapeo](https://mapeo.world), [Cabal](http://cabal.chat), [Wireline](https://wireline.io), and [Peermaps](https://peermaps.org).

**Comment by karissa** [comment]
definitely should talk about beaker blue! @pfrazee

**Comment by RangerMauve** [comment]
Would anyone have time to do a blog post or something on Kappa-Core? It might be best to put it all in one place so people can stumble across it in the future.

**Comment by pfrazee** [comment]
Happy to put together a short write-up @RangerMauve for blue

**Comment by brechtcs** [comment]
I've released a version of [dterm](https://github.com/dfurl/dterm/releases/tag/v0.2.0) today that I think is ready for Actual People™ to start using it. It's not exactly feature complete yet, but all the features that are there now should be stable enough to keep them backwards compatible.

**Comment by RangerMauve** [comment]
@brechtcs That looks really cool! Do you have a blog post or something explaining what it does?

**Comment by pfrazee** [comment]
@brechtcs awesome! We should do a call and talk about this

**Comment by noffle** [comment]
@RangerMauve what would you be looking for re kappa-core? I've been meaning to do a detailed intro, but I'm not sure when I'll have time.

**Comment by RangerMauve** [comment]
@noffle I was thinking of sending out the newsletter at the end of the month, but if next month, or some other time in the future would be better for you that works too. It's not like we're in a rush to publish anything specific. :P

**Comment by brechtcs** [comment]
@RangerMauve The goals of dterm are largely a continuation of @pfrazee's [original objectives](https://pfrazee.hashbase.io/blog/reimagining-the-browser-as-a-network-os). Things have evolved a bit since then though: from the builtin `term://` protocol in Beaker, to [Webterm](https://github.com/pfrazee/webterm), then forked into dterm. A basic write-up of those changes might be what you're looking for?

@pfrazee A call would definitely be interesting yes. My schedule is a bit tight right now for family reasons, but I'll try to figure things out and drop you an email later this week.

**Comment by ylebre** [comment]
@RangerMauve here are some links to what we are working on. The code is either MIT or GPL licensed.

**dat://designblocks.sharespace.nl**
An editable site that also contains a simple page-builder-like tool to quickly change the looks.
The design itself uses the Froala design blocks (from back in the day when the blocks were still GPL licensed - sadly they have a more restrictive license now).

**dat://blog-ylebre.sharespace.nl/helloworld/**
A quick intro into an editable site in BeakerBrowser, using SimplyEdit as the editor.

**dat://blog-ylebre.sharespace.nl/**
An introduction to the concepts we are using within SimplyEdit.

**Comment by RangerMauve** [comment]
@pfrazee @brechtcs Any updates on blog posts for Blue / dterm? We can put it in the June newsletter if it's not ready yet. :)

**Comment by RangerMauve** [comment]
Anybody against me plugging [local-first cyberspace](https://github.com/RangerMauve/local-first-cyberspace)? 😜

**Comment by todrobbins** [comment]
Did we send off the May newsletter or will this content be included in the June newsletter?

**Comment by RangerMauve** [comment]
I'm just drafting up the may newsletter today. Hoping to get it reviewed and released tomorrow. :)

**Comment by pfrazee** [comment]
I *may* have an unwalled garden blogpost this week but no promises. Don't wait on me.

**Comment by brechtcs** [comment]
@RangerMauve I'm considering making some non-trivial breaking changes to dterm in v0.3 for security reasons (see issue https://github.com/dfurl/dterm/issues/8), so will postpone the announcement post until that's resolved.

**Comment by cblgh** [comment]
the newsletter for historical reference: https://mailchi.mp/652b2b95da41/some-things-that-happened-during-may-in-the-dat-ecosystem
